firebase.json

```
@@ -0,0 +1,19 @@
+{
+  "hosting": {
+    "public": "dist",
+    "ignore": [
+      "firebase.json",
+      "**/.*",
+      "**/node_modules/**"
+    ],
+    "rewrites": [
+      {
+        "source": "**",
+        "destination": "/index.html"
```

やりたいこと理解しました。destinationの設定は必要なく、代わりに関数の設定が必要です！appのところは、実装している関数名に合わせる必要があります！

```suggestion
        "function": "app"
```